js_modules/dagster-ui/packages/ui-core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -210,6 +218,8 @@ export const AssetsCatalogTable = ({
       searchPath={searchPath}
       displayPathForAsset={displayPathForAsset}
       requery={(_) => [{query: ASSET_CATALOG_TABLE_QUERY, fetchPolicy: 'no-cache'}]}
+      computeKindFilter={computeKindFilter}
```

Comment:
Hmm out of scope for this PR but that requery seems wrong. We should remove that